src/components/accordion/accordion.js

```diff
@@ -0,0 +1,127 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Collapse } from 'react-collapse';
+import cx from 'classnames';
+// import Icon from '../icon';
+import Icon from '@bit/aabdaab.cw-components.global.icon';
+
+import dropdownArrow from './assets/dropdown-arrow.svg';
+import styles from './accordion-styles.scss';
+
+/**
+ * Table component to show nested data structures
+ */
+class Accordion extends PureComponent {
+  render() {
+    const {
+      data,
+      theme,
+      handleOnClick,
+      openSlug,
+      children,
+      isChild,
+      hasNestedCollapse
+    } = this.props;
+    return (
+      <div className={theme.wrapper}>
+        {data &&
+          data.length > 0 &&
+          data.map((section, index) => {
+            let isOpen = index === 0;
+            if (openSlug) {
+              if (openSlug !== 'none') {
+                const isActiveInResults = data.some(d => d.slug === openSlug);
+                isOpen =
+                  openSlug === section.slug ||
+                  (index === 0 && !isActiveInResults);
+              } else {
+                isOpen = false;
+              }
+            }
+            const title = section.parent
+              ? `${section.parent.name} | ${section.title}`
+              : section.title;
+            return (
+              <section
+                key={`${section.slug}-${section.title}`}
+                className={cx(styles.accordion, theme.accordion)}
```

Comment:
the `theme.accordion` is not going to throw an error if theme is not passed? shouldn't the defaultProps be set to
```
theme: {
  accordion: ''
  ...
}
```
?

Reply:
nah, it's fine. I confused it with the error when you don't have the defaultProps set to an empty object. should be good!

Reply:
I don't think it is worth it because is going to try to access a property that doesn't exist so className library will receive an undefined and won't include it, what is the same of the empty string.